examples/pvtol-lqr.py

```diff
@@ -21,35 +20,37 @@
 #
 
 # System parameters
-m = 4;				# mass of aircraft
-J = 0.0475;			# inertia around pitch axis
-r = 0.25;			# distance to center of force
-g = 9.8;			# gravitational constant
-c = 0.05;	 		# damping factor (estimated)
+m = 4  # mass of aircraft
```

Comment:
My personal preference is to have the comments lined up; makes them easier to read.

Reply:
Agreed.

Reply:
Turns out there were still tab characters all over the place, hence the misalignment.  Someone has their text editor set to allow tabs still maybe?